rust/main/hyperlane-base/src/contract_sync/cursors/sequence_aware/backward.rs

```diff
@@ -161,6 +161,10 @@ impl<T: Debug + Clone + Sync + Send + Indexable + 'static> BackwardSequenceAware
 
                 self.current_indexing_snapshot = self.last_indexed_snapshot.previous_target();
 
+                // Update metrics during fast-forward (actually backward in this case) so that
+                // the metrics do not stuck on the last indexed sequence.
+                self.update_metrics().await;
```

Comment:
just realizing now that `update_metrics` doesn't actually need to be async, wonder if you could change as a drive by?

Reply:
Fixed